### PR TITLE
Realease

### PR DESCRIPTION
**Pull request by redben**
- Reverts loomnetwork/dashboard#1334
- fix Gateway version on production

### CODE REVIEW

File: src/components/gateway/WithdrawConfirmed.vue [hunk 1]
```diff
@@ -2,9 +2,12 @@
   <b-modal lazy id="withdraw-confirmed" v-model="visible" :title="title">
     <section v-if="status === 'default'">
       <b-container fluid>
-        <div class="lead">
+        <div v-if="chain !== 'binance'" class="lead">
           <p>{{$t('components.gateway.confirm_withdrawal_modal.confirm_withdrawl', {chain,amount, token: this.symbol})}}</p>
         </div>
+        <div v-else>
+           <p>Fund withdrawn to Binance</p>
```
Comment (enlight) [inline src/components/gateway/WithdrawConfirmed.vue]: -> `Funds withdrawn to Binance`

File: src/index.ts [hunk 2]
```diff
@@ -60,9 +59,19 @@ export default new Vue({
...
+
+      dashboardStore.setEnvs(multipleEnvs)
+
+      if (userSelected) {
+        // if user has already selected a config before
+        const selectedEnv = multipleEnvs.find((env) => env.name === userSelected)!
+        await dashboardStore.setEnv(selectedEnv)
+      } else {
+        // default
+        await dashboardStore.setEnv(extDev)
```
Comment (enlight) [inline src/index.ts]: Why does this default to Extdev? Shouldn't it default to Basechain?

File: src/components/gateway/WithdrawConfirmed.vue (hunk 1 above)
Comment (enlight) [reply src/components/gateway/WithdrawConfirmed.vue]: Can fix this later.

File: src/index.ts (hunk 2 above)
Comment (redben) [reply src/index.ts]: It default to ext-dev when not on production.

**Pull request merged**